### DISCUSSION TITLE
Is it possible to deploy it on Win?

### DISCUSSION THREAD

**Discussion by GitZinger** [shadow/shadow#729] [Q&A] [answered]
**State your question concisely**
Is it possible to deploy it on Win?
**Describe your current understanding of Shadow as it relates to your question**
tried very hard to build on linux. there are ton of errors.
**Describe which parts you are unsure about**
there is no description about it

**Comment by robgjansen** [comment, marked as answer]
Sorry, but support for windows is not planned at the moment. (We need to prioritize development effort and we have lots of work even to properly support Linux.) See also my related comment about your environment [here](https://github.com/shadow/shadow/issues/728#issuecomment-594728188), where I suggest trying Shadow on Ubuntu 18.04 (Github's default test environment.)
